src/Compilers/CSharp/Test/Syntax/Parsing/ParserErrorMessageTests.cs

```diff
@@ -5594,6 +5594,96 @@ class TestClass { }";
             N(SyntaxKind.EndOfFileToken);
         }
 
+        [Fact]
+        public void RefExtensionMethodsNotSupportedBefore7_2_RefReadOnlySyntax()
```

Comment:
>RefExtensionMethodsNotSupportedBefore7_2_RefReadOnlySyntax [](start = 20, length = 58)

Aren't all the new tests in this file essentially duplicates of what we have in RefExtensionMethodsTests.cs? Please remove.